### PR TITLE
kops tweaks

### PR DESCRIPTION
**Pull request by jaxesn**
- support kops 1.19 change around kubecfg admin exporting
 - fix tags for prerelease builds
 - fix duplicate kublet config in template

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: development/kops/create_values_yaml.sh [hunk 1]
```diff
@@ -26,62 +26,79 @@ if [ -f "./${KOPS_CLUSTER_NAME}/values.yaml" ]; then
     exit 1
 fi
 
-
 function get_container_latest_tag() {
     REPOSITORY_NAME="${1}"
-    DEFAULT_TAG="${2}"
+    VERSION="${2}"
```
Comment (TerryHowe) [inline development/kops/create_values_yaml.sh:31]: why are these changes showing up again?

File: development/kops/eks-d.tpl [hunk 2]
```diff
@@ -111,7 +110,7 @@ metadata:
   name: control-plane-{{.awsRegion}}a
 spec:
   image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20201026
-  machineType: t3.medium
+  machineType: m5.xlarge
```
Comment (TerryHowe) [inline development/kops/eks-d.tpl:113]: We probably don't want to check this in, but the other two changes look good.

File: development/kops/create_cluster.sh [hunk 3]
```diff
@@ -19,4 +19,4 @@ BASEDIR=$(dirname "$0")
 source ${BASEDIR}/set_environment.sh
 $PREFLIGHT_CHECK_PASSED || exit 1
 
-${KOPS} update cluster --name ${KOPS_CLUSTER_NAME} --yes
+${KOPS} update cluster --admin --name ${KOPS_CLUSTER_NAME} --yes
```
Comment (jaxesn) [inline development/kops/create_cluster.sh:22]: I want to test this with the kops 1.18

File: development/kops/eks-d.tpl (hunk 2 above)
Comment (jaxesn) [reply development/kops/eks-d.tpl:115]: Watching the run, not sure this is making much difference, Ill probably revert this or make it configurable if neccessary.

**Review by TerryHowe — APPROVED** [review]

**Pull request merged**